build.sbt

```diff
@@ -18,11 +18,17 @@ ThisBuild / githubWorkflowBuild := Seq(
     name = Some("Build native assembly")
   )
 )
-ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "graalvm-ce-java11@")
-ThisBuild / githubWorkflowOSes := Seq("ubuntu-latest", "macos-latest")
-ThisBuild / githubWorkflowBuildMatrixExclusions := Seq(MatrixExclude(Map("os" -> "macos-latest", "java" -> "adopt@1.8")))
+
+val PrimaryOS = "ubuntu-latest"
+ThisBuild / githubWorkflowOSes := Seq(PrimaryOS)
+ThisBuild / githubWorkflowEnv += ("JABBA_INDEX" -> "https://github.com/typelevel/jdk-index/raw/main/index.json")
```

Comment:
Is Jabba index needed here if we use `JavaSpec` from the Github Actions plugin? I'm asking for myself, some projects are waiting for the same kind of job :)

Reply:
As far as I can tell, this is needed to support versions not yet [available on GitHub Actions](https://github.com/actions/setup-java#supported-distributions).
Here I think this's required for graalvm, but I may be wrong :)

Reply:
I mean that `sbt-github-actions` itself doesn't contain Jabba index and still has the graalvm https://github.com/djspiewak/sbt-github-actions/blob/main/.github/workflows/ci.yml#L26. But for sure, I could be missing something obvious.

Reply:
No,there's no more Jabba.

Reply:
```suggestion
```